elrond.md

```diff
@@ -737,14 +746,18 @@ Only take the next step once both the Elrond node and Wasm are done executing.
                       | deployTxAux ( Bytes, Int, ModuleDecl, List, Int, Int )   [klabel(deployTxAux), symbol]
  // ----------------------------------------------------------------------------------------------------------
     rule <k> deployTx(FROM, VALUE, MODULE, ARGS, GASLIMIT, GASPRICE)
-	     => deployTxAux(#address2Bytes(FROM), VALUE, MODULE, ARGS, GASLIMIT, GASPRICE) ... </k>
+	      => deployTxAux(#address2Bytes(FROM), VALUE, MODULE, ARGS, GASLIMIT, GASPRICE) ... </k>
```

Comment:
Arrow head (`>`) should align with end of `<k>` tag.

Reply:
fixed